OrbitGl/CaptureWindow.cpp

```diff
@@ -529,6 +524,7 @@ void CaptureWindow::KeyPressed(unsigned int a_KeyCode, bool a_Ctrl,
         break;
       case 'X':
         GOrbitApp->ToggleCapture();
+        m_DrawHelp = false;
```

Comment:
I'm not sure if this (removing the help) is desired if the user explicitly toggled the help on by pressing 'H'. Not a huge concern, though.

Reply:
The rationale was that once a users are ready to capture, they have read the short help section and don't need it anymore.  It also makes it so that we don't have to keep track of whether the help popup was triggered by a user or if it was there by default.